cdap-docs/application-templates/source/etl/templates/sinks/batch/s3avro.rst

```diff
@@ -0,0 +1,58 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+==========================
+Sinks: Batch: S3Avro
+==========================
+
+.. rubric:: Description
+
+A batch sink for a to write to Amazon S3 in Avro format. 
```

Comment:
seems like you're missing a word:
"A batch sink for a <MISSING> to write to..."